_conferences/2024/call.html

```diff
@@ -0,0 +1,167 @@
+
+    <h4 id="topics">Topics</h4>
+
+    <p>The conference welcomes contributions from all those who are developing or applying music encodings in their work and research. Topics include, but are not limited to:</p>
+    <ul>
+        <li>data structures for music encoding</li>
+        <li>music encoding standardisation</li>
+        <li>music encoding interoperability / universality</li>
+        <li>methodologies for encoding, music editing, description and analysis</li>
+        <li>computational analysis of encoded music</li>
+        <li>rendering of symbolic music data in audio and graphical forms</li>
+        <li>conceptual encoding of relationships between multimodal music forms (e.g. symbolic music data, encoded text, facsimile images, audio)</li>
+        <li>capture, interchange, and re-purposing of musical data and metadata</li>
+        <li>ontologies, authority files, and linked data in music encoding and description</li>
+        <li>(symbolic) music information retrieval using music encoding</li>
+        <li>evaluation of music encodings</li>
+        <li>best practice in approaches to music encoding</li>
+    </ul>
+
+    <p>and the use or application of music encodings in:</p>
+    <ul>
+        <li>music theory and analysis</li>
+        <li>digital musicology and, more broadly, digital humanities</li>
+        <li>digital editions</li>
+        <li>music digital libraries</li>
+        <li>bibliographies and bibliographic studies</li>
+        <li>catalogues and collection management</li>
+        <li>composition</li>
+        <li>performance</li>
+        <li>teaching and learning</li>
+        <li>search and browsing</li>
+        <li>multimedia music presentation, exploration, and exhibition</li>
+        <li>machine learning approaches.</li>
+    </ul>
+
+
+    <h4 id="submissions">Submissions</h4>
+    <p> MEC 2024 will be accepting submissions in the following formats for the main conference programme (all submissions should include a short abstract, keywords and bio):</p>
+    <p>
+<ul>
+  <li><strong>Long papers:</strong> abstract 2-4 pages (1000-2000 words).</li>
+  <p>Long papers are expected to present overviews or detail specific aspects of ongoing or completed projects, detailed case-studies or elaborated perspectives on best practices in the field, or provide other reports on topics relevant to the conference.</p>
```

Comment:
@mfeustle it doesn't look like the tags are nested correctly. Please see the note about the `li` element above.